### DISCUSSION TITLE
CSS for MyST outside of jupyter-book?

### DISCUSSION THREAD

**Discussion by paugier** [executablebooks/meta#823] [General]
I tried to investigate how MyST could be used outside of jupyter-book. Different applications

- [single page conversion without sphinx](https://myst-parser.readthedocs.io/en/latest/docutils.html)
- Pelican MyST reader (see https://github.com/ashwinvis/myst-reader)
- Conversion of one .md file into a static html presentation (for example with nbconvert or [markdown-to-presentation](https://pypi.org/project/markdown-to-presentation/), see https://github.com/orgs/executablebooks/discussions/772)

For these different applications, good CSS compatible with MyST is of course very important. However, I don't see where to get such CSS files. For jupyter-book, I see that the styles are defined in

```
$ ls book/_build/html/_static/styles/
pydata-sphinx-theme.css  sphinx-book-theme.css  theme.css
```

- For the single page conversion with only docutils, the css are embeded in the html file and the style are much less appealing.
- for the Pelican reader the CSS definitions corresponding to MyST classes are just missing

So how what is done in pydata-sphinx-theme.css and sphinx-book-theme.css can be reused / reproduced for other applications of MyST?

**Comment by paugier** [comment]
I'd like to try to work on a project to build a simpler converter from a myst.md file to a reveal.js presentation (some notes here https://github.com/paugier/myst-pres). Unfortunately, this question on CSS seems to be one of the main blocker for this project. Any help would be greatly appreciated!

**Reply by rowanc1** [reply]
Super interested in presentations! We are doing some work on `juptyerlab-myst` ([repo](https://github.com/executablebooks/jupyterlab-myst)) at the moment which has most of the MyST functionality. In this case we are a bit more than just the HTML output (e.g. it has interactive hover-references, and some dynamic content).

If you are interested in that route (e.g. using React), I can point to a lot more resources and ways of doing this.
